manifests/generated/rbac-kubevirt.authorization.k8s.yaml.in

```diff
@@ -285,10 +285,17 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - secrets
   - persistentvolumeclaims
   verbs:
   - get
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - create
+  - update
```

Comment:
I think we just need 'get' and 'create'

also 'create' could be limited to the namespace kubevirt infra is running in.  I don't think that's a big deal though as long as we don't have permissions to 'update' in any namespace.

Reply:
Reverted the cluster-wide  permissions to its old state (*just* get), and added create for the install namespace.